tutorial/source/ss-vae.ipynb

```diff
@@ -104,33 +104,33 @@
     "    # sample z from the prior \n",
     "    prior_mu = Variable(torch.zeros([batch_size, z_dim]))\n",
     "    prior_sigma = Variable(torch.ones([batch_size, z_dim]))\n",
-    "    zs = pyro.sample(\"z\", dist.normal, prior_mu, prior_sigma)\n",
+    "    zs = pyro.sample(\"z\", dist.Normal(prior_mu, prior_sigma))\n",
     "\n",
     "    # if the label y is observed, sample from the prior.\n",
     "    # otherwise, observe the value \n",
     "    alpha_prior = Variable(torch.ones([batch_size, 10]) / (10.))\n",
     "    if ys is None:\n",
-    "        ys = pyro.sample(\"y\", dist.one_hot_categorical, alpha_prior)\n",
+    "        ys = pyro.sample(\"y\", dist.One_hot_categorical(alpha_prior))\n",
```

Comment:
Should be CamelCase: `dist.OneHotCategorical`

here and elsewhere in this PR

Reply:
good catch, ill fix that